src/index.test.js

```diff
@@ -0,0 +1,109 @@
+  });
+
+  test('check for the presence of the restart button', () => {
+    document.body.innerHTML = `
+    <p class="notification"></p>
+    <div class="restart-div"><button class="restart-btn">Restart</button></div>
+    `;
+    const restartBtn = document.getElementsByTagName('button');
+    expect(restartBtn).toBeTruthy();
+  });
+
+  test('check when there is a draw', () => {
+    document.body.innerHTML = `
+    <p class="notification"></p>
+    <div class="restart-div"><button class="restart-btn" style="display:none;">Restart</button></div>
+    `;
+
+    const playerFunc = require('./index');
+    const { Gameboard, game, Player } = playerFunc;
+    const currentPlayer = Player('Mike', 'X');
+    Gameboard.table = 'XOXXOOOXO'.split('');
+
+    const notification = document.querySelector('.notification');
+    const restartBtn = document.querySelector('.restart-btn');
+    expect(game.draw()).toBe(true);
+  });
+
+  test('check when there is a win', () => {
+    const playerFunc = require('./index');
+    const { Gameboard, game, Player } = playerFunc;
+    const currentPlayer = Player('Roodz', 'X');
+    Gameboard.table = ['X', 'X', 'X', 'O', 'O', 'X', 6, 7, 8];
+    expect(game.checkWin()).toBe(true);
+  });
+
+  test('check when the player1 has won', () => {
+     document.body.innerHTML = `
+    <p class="notification"></p>
+    <div class="restart-div"><button class="restart-btn" style="display:none;">Restart</button></div>
+    `;
+    const playerFunc = require('./index');
+    const { Gameboard, game, Player } = playerFunc;
+    const notif = document.getElementsByClassName("notification");
+    let player1 = Player("Roodz", "X");
+    let player2 = Player("Marvellous", "O");
+    const currentPlayer = player1;
+    for(let i = 0; i < Gameboard.table.length; i++) {
+      if (i % 2 == 0) {
+        Gameboard.table[i] = player1.marker;
+      } else {
+        Gameboard.table[i] = player2.marker;
+      }
+    }
+    notif.textContent = `${currentPlayer.name} wins`;
+    expect(Gameboard.table).toEqual([
+      'X', 'O', 'X',
+      'O', 'X', 'O',
+      'X', 'O', 'X'
+    ]);
+  });
+
+  test("Check each players' turn", () => {
+      document.body.innerHTML = `
+    <p class="notification"></p>
+    <div class="restart-div"><button class="restart-btn" style="display:none;">Restart</button></div>
+    `;
+    const playerFunc = require('./index');
+    const { game, Player } = playerFunc;
+    const notif = document.getElementsByClassName("notification");
+    let player1 = Player("Roodz", "X");
+    let player2 = Player("Marvellous", "O");
+    const players = [player1, player2];
+    let currentPlayer = player1;
+    function changeCurrentPlayer() {
+      const [player1, player2] = players;
+      if (currentPlayer === player1) {
+        currentPlayer = player2;
+      } else {
+        currentPlayer = player1;
+      }
+    }
+    changeCurrentPlayer();
+    expect(currentPlayer).toEqual(player2);
+  });
+});
```

Comment:
Why don't you directly use the `game#changePlayer` ?
Test it directly!!

You imported the `game` from `index.js` but didn't use it. Why?
Instantiate it, make two or three turn and check who the current player is.